src/hooks/useSelectTarget.js

```diff
@@ -0,0 +1,36 @@
+import { useEffect, useCallback } from 'react';
+import _ from 'lodash';
```

Comment:
Here you import all lodash but you are using just `find` function, you can do:

```suggestion
import find from 'lodash/find';
```
and use find directly